profiles/bicycle.lua

```diff
@@ -320,6 +320,7 @@ function way_function (way, result)
   local foot_backward = way:get_value_by_key("foot:backward")
   local bicycle = way:get_value_by_key("bicycle")
 
+  local allowed_way_type = false
```

Comment:
what about pushing_possible, since that's why it's needed: if tags like oneway means we can't bike, we need to check if we can push the bike. instead of rechecking all the way types we see if pushing_possible is set.